### ISSUE TITLE
Guides governance and process

### ISSUE THREAD

**Issue by coreycaitlin** [18F/archived-guides#33]
Given the proliferation of guides, it's easy for the [guides page](https://pages.18f.gov/guides/) to be overwhelming. One thing that would help is to introduce a very light governance process so that we don't add guides to the left-nav until they're ready for broader reading. That is, we shouldn't be linking to guides that are very rough drafts or WIP from this page.

It seems fine for guides to mainly exist on GitHub while in progress.

Any objections? What would we want that lightweight process to look like? Should it include any approvals, or just a note about the guide being at v1 + names of people or guilds who will contribute to maintaining it?

**Comment by wslack** [comment]
I think this is our current tentative process, albeit informal? We didn't link the Partnership Playbook immediately. I would defer to @mtorres253 and @mbland as to a governance process to get listed on the main guides sidebar.

**Comment by mbland** [comment]
Yes, something _very_ light like this should get codified. I think we'll need to get resolution on #31 and #32 first, though.
